src/main/scala/Game.scala

```diff
@@ -10,18 +12,27 @@ import Move._
 //   Rock beats Scissors, Paper beats Rock, Scissors beats Paper
 
 object Game {
-  private def randomMove(): Move = moves(Random.nextInt(moves.length))
+  private def randomMove(): Move = {
+    val moves = CaseEnumSerialization[Move].values.toIndexedSeq
```

Comment:
suggestion: `toList` is a bit more common

```suggestion
    val moves = CaseEnumSerialization[Move].values.toList
```